Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -644,6 +644,13 @@ lemma sum_cons' [AddCommMonoid M] [AddCommMonoid N] (n : ℕ) (σ : Fin n →₀
   simp_rw [Fin.sum_univ_succ, cons_zero, cons_succ]
   congr
 
+@[to_additive]
+lemma prod_mul_eq_prod_mul_of_exists [DecidableEq α] [Zero M] [CommMonoid N]
+    {f : α →₀ M} {g : α → M → N} {n₁ n₂ : N}
+    (h : ∃ a ∈ f.support, g a (f a) * n₁ = g a (f a) * n₂) :
```

Comment:
Should `h` be unbundled? (That is `(a) (ha : f.support) (ha' : g a (f a) * n₁ = g a (f a) * n₂)`)